charts/sonarqube/templates/install-plugins.yaml

```diff
@@ -9,21 +9,12 @@ metadata:
     heritage: {{ .Release.Service }}
 data:
   install_plugins.sh: |-
-    {{- if .Values.plugins.httpProxy }}
-    export http_proxy={{ .Values.plugins.httpProxy }}
-    {{- end }}
-    {{- if .Values.plugins.httpsProxy }}
-    export https_proxy={{ .Values.plugins.httpsProxy }}
-    {{- end }}
-    {{- if .Values.plugins.noProxy }}
-    export no_proxy={{ .Values.plugins.noProxy }}
-    {{- end }}
     {{- if .Values.plugins.install }}
       [ -e {{ .Values.sonarqubeFolder }}/extensions/downloads/* ] && rm {{ .Values.sonarqubeFolder }}/extensions/downloads/*
-      {{ range $index, $val := .Values.plugins.install }}
-      echo {{ $val | quote }} >> {{ $.Values.sonarqubeFolder }}/extensions/downloads/list{{ end }}
-      cat {{ .Values.sonarqubeFolder }}/extensions/downloads/list | xargs -n 1 -P 8 wget --directory-prefix {{ .Values.sonarqubeFolder }}/extensions/downloads --no-verbose{{ if .Values.plugins.noCheckCertificate }} --no-check-certificate{{ end }}
-      rm {{ .Values.sonarqubeFolder }}/extensions/downloads/list
+      cd {{ .Values.sonarqubeFolder }}/extensions/downloads
+      {{- range $index, $val := .Values.plugins.install }}
+      curl {{ if $.Values.plugins.noCheckCertificate }}--insecure{{ end }} -fsSLO {{ $val | quote }}
```

Comment:
i know that this has previously also been downloaded without checking certificates, but now reading it again i think it might make sense to leave the option, but now be insecure by default.
can you take care about that in this PR? else i will handle this separated

Reply:
I am a bit confused as to what you are requesting, do you want me to:
* change it so it always uses `--insecure`
* checking if this now is defaulting to `--insecure` (it should not right now unless `.Values.plugins.noCheckCertificate` is set to true)
* my opinion on defaulting too `--insecure`, which I am no fan of due to security concerns NGL
* update documentation

I am open for updating it whatever way you want me to, just require some clarification :wink:

Reply:
oh sorry i missed that if statement :see_no_evil: this was exactly what i wanted.